telecom colorcode.h

```diff
@@ -0,0 +1,30 @@
+#ifndef DEFINE_MAIN_ONCE
+#define DEFINE_MAIN_ONCE
+
+#include <stdio.h>
+#include <assert.h>
+
+enum MajorColor {WHITE, RED, BLACK, YELLOW, VIOLET};
+enum MinorColor {BLUE, ORANGE, GREEN, BROWN, SLATE};
+
+typedef struct {
+    enum MajorColor majorColor;
+    enum MinorColor minorColor;
+} ColorPair;
+
+ColorPair GetColorFromPairNumber(int pairNumber);
+
+void testNumberToPair(int pairNumber,
```

Comment:
Declaration of test code is part of production code files. Production code needs to be kept separately from test code so that test code can be excluded during shipment of software to field.
Separation also helps in maintenance.